src/components/i18nTextarea.vue

```diff
@@ -212,9 +200,25 @@ export default {
 			}
 			this.addTab(lang)
 		},
+		"$store.state.languages": function(languages) {
+			for (const lang in languages) {
+				if(languages[lang]) {
```

Comment:
Add space here after if
```suggestion
				if(languages[lang]) {
```